ui/src/routes/Tool/Tool.js

```diff
@@ -1,17 +1,30 @@
 import React, { Component } from 'react'
 import { browserHistory } from 'react-router'
+import PageFocusSection from '../../components/PageFocusSection'
+import LessonModal from '../../components/LessonModal'
 import { log } from '../../utilities'
 
-// import '../../styles/buttons.css'
 import '../../styles/components/c-header.css'
 import '../../styles/components/c-activity.css'
 import '../../styles/components/c-modal.css'
 
-class Subjects extends Component {
+class Tool extends Component {
   constructor (props) {
     super(props)
     this.state = {
-      showModal: false
+      showModal: false,
+      skiplinks: [{
```

Comment:
Do you need to define skipLinks here? I think it's already defined in PageFocusSection...